### ISSUE TITLE
Images from old notes don't show up anymore (but new one do)

### ISSUE THREAD

**Issue by gustavklopp** [laurent22/joplin#7939]
As in the title: Images from old notes don't show up anymore (but new one do)
The "old" notes are maybe 6 months ago but I don't know exacly what the cut-off dates where the images can't be loaded anymore. 

They know appear like this:
![image](https://user-images.githubusercontent.com/6438275/225826819-5ce4cc13-e1bb-40b9-8c4e-c032e563f751.png)

This is what is in the edit mode:
![image](https://user-images.githubusercontent.com/6438275/225826897-edcdacba-0bb5-41b0-abde-fba541a89d53.png)

I've checked in the folder where (if I am reasoning correctly) Joplin stores the images:
` ./.config/joplin-desktop/resources/` 
And I've found no picture for example:
`5d31ceba9bbb49f5bd4dc5562f04b31e` is found nowhere.

So where did all the pictures from my old notes went?
I erased nothing myself, so my guess is that Joplin did it. Maybe during an update?

## Environment

Joplin version: Joplin 2.9.17 (prod, linux) Revision: a84a8e7
Platform: Linux Manjaro

**Comment by laurent22** [comment]
Did you use any resource cleaner script or plugin? What plugins do you use?

**Comment by gustavklopp** [comment]
Thanks for your answer. I don't use any resource cleaner for what I know.
The only plugin is Outline.
Can I check in the Joplin database somewhere if this file is recorded?

**Comment by gustavklopp** [comment]
Joplin can't show my pictures again! 
Here's what I get now for a Note that I created about 2 weeks ago:
![image](https://user-images.githubusercontent.com/6438275/233587721-e89c48c1-7275-4fb1-852f-759e7d24fdfe.png)
And the code editor shows:
![image](https://user-images.githubusercontent.com/6438275/233587933-671a0e23-4c42-435a-b9dd-fa6acabfa04c.png)

I've searched in my folder where the new images are usually stored: `./.config/joplin-desktop/resources/`
`79b6443f21eb4ea3983fe6e8a82d24e3.png` does exist though.
But it's not showing up in Joplin.
